packages/web/projects/web-components/src/components/buttons/button/button.component.ts

```diff
@@ -38,6 +39,8 @@ type iconPos = 'left' | 'right';
     '[class.freud-btn-size-small]': `size === 'sm'`,
     '[class.freud-btn-size-medium]': `size === 'md'`,
     '[class.freud-btn-size-large]': `size === 'lg'`,
+
+    '[class.freud-btn-full-width]': `fullWidth ? 'freud-btn-full-width' : ''`,
```

Comment:
```suggestion
    '[class.freud-btn-full-width]': 'fullWidth',
```

Aqui acredito que dê para simplificar para ficar assim.

Reply:
Realmente! Vou modificar

Reply:
Quando subir o commit com essa alteração, me avisa.

Reply:
Acabei de subir